e2eshark/onnx/models/opt-125M-awq/model.py

```diff
@@ -16,34 +16,92 @@
 # to form runmodel.py in the rundirectory which is then taken
 # through flow
 
-# set some variable parameters (dynamic dims)
-batch_size = 3 
-sequence_length = 7 
-past_sequence_length = 0 # setting this parameter to anything other than 0 results in an error
+# If you want to run the test with a static model.onnx, set run_as_static = True
+run_as_static = False
```

Comment:
Please clarify what specifically is static or dynamic about the model.

Also, is this worth making configurable via `argparse` (https://docs.python.org/3/library/argparse.html)?